Sources/AEPOptimize/Offer.swift

```diff
@@ -68,6 +72,8 @@ public class Offer: NSObject, Codable {
         // Try and decode format, if present. Target response doesn't contain etag, so setting the default value to empty string.
         etag = try container.decodeIfPresent(String.self, forKey: .etag) ?? ""
 
+        score = try container.decodeIfPresent(Int.self, forKey: .score) ?? 0
```

Comment:
since the score is only returned when a ranking method is present, is `0` a valid "non value"?  or should we consider making `score` optional?

Reply:
Thanks @sbenedicadb. 0 is the default Offer priority. I didn't make it optional, because we expect it to always be present (this is currently under discussion).